commands/reset-giveaway.js

```diff
@@ -0,0 +1,31 @@
+const { CommandInteraction, Util } = require('discord.js');
```

Comment:
This needs to be a SlashCommandBuilder NOT CommandInteraction

Reply:
This is because of how the deploy commands reads the file types for the discord API